yunwoo/programmers-1/하샤드-수.js

```diff
@@ -0,0 +1,6 @@
+function solution(x) {
+  const result = Array.from(String(x)).reduce((acc, cur) => {
+    return Number(cur) + acc;
+  }, 0);
+  return x % result === 0 ? true : false;
+}
```

Comment:
```suggestion
function solution(x) {
  const result = Array.from(String(x)).reduce((acc, cur) => Number(cur) + acc, 0);
  
  return x % result === 0 ? true : false;
}
```
- Array.from을 이용해서 배열로 변환 후에 reduce 해주는 방법 좋았습니다 👍
- 굳이 중괄호 하고 리턴하지 않고 화살표 함수에서 중괄호 제거하고 바로 리턴이 되기 때문에 해당 방법도 참고해주시면 좋을 것 같습니다

Reply:
- 헉 한줄로 가능하군요..! 다음에 사용할때는 고려해서 사용하겠습니다 감사합니다👍